lib/mbta.js

```diff
@@ -1,9 +1,100 @@
 'use strict'
 
 // Code here.
+// pseudo Code
+// create an object with arrays for each of thew SubwayLines
+const mbta = {
+  const redLine = [
+    'South Station',
+    'Park Street',
+    'Kendall',
+    'Central',
+    'Harvard',
+    'Porter',
+    'Davis',
+    'Alewife'
+  ]
 
-const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
+  const greenLine = [
+    'Government Center',
+    'Park Street',
+    'Boylston',
+    'Arlington',
+    'Copley',
+    'Hynes',
+    'Kenmore'
+  ]
 
+  const orangeLine = [
+    'North Station',
+    'Haymarket',
+    'Park Street',
+    'State',
+    'Downtown Crossing',
+    'Chinatown',
+    'Back Bay',
+    'Forest Hills'
+  ]
+}
```

Comment:
Your `mbta` object should look something like this: `const subwayLines = {
  Red: [
    'South Station',
    'Park Street',
    'Kendall',
    'Central',
    'Harvard',
    'Porter',
    'Davis',
    'Alewife'
  ],
  Green: [
    'Government Center',
    'Park Street',
    'Boylston',
    'Arlington',
    'Copley',
    'Hynes',
    'Kenmore'
  ],
  Orange: [
    'North Station',
    'Haymarket',
    'Park Street',
    'State',
    'Downtown Crossing',
    'Chinatown',
    'Back Bay',
    'Forest Hills'
  ]
}`